fastlane/Fastfile

```diff
@@ -299,6 +299,34 @@ end
 
 lane :release do
   desc "Make github release for the latest tag and make builds"
+  ANDROID_ASSETS_FOLDER = Dir.children( "../android/app/src/main/assets/camera" )
```

Comment:
This doesn't need to be a constant since it's not used outside of this context. You can just do something like `Dir.children( "../android/app/src/main/assets/camera" ).each do | file |`.